internal/provisioner/controller/gatewayclass.go

```diff
@@ -22,6 +22,7 @@ import (
 
 	"github.com/go-logr/logr"
 	corev1 "k8s.io/api/core/v1"
+	apiextensions_v1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
Unify to ...vN or ..._vN ?

Reply:
Good shout, though I think we'll wait to do a bulk change of import aliases in https://github.com/projectcontour/contour/pull/6094